CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 10.22
+* Bump content-api-models (refactored circe macros)
```

Comment:
This shouldn't say `bump content-api-models` as the consumer doesn't know what that means.